modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/form_builder_field_settings_sidebar.js

```diff
@@ -55,6 +55,20 @@ AUI.add(
 						instance._eventHandlers = eventHandlers;
 					},
 
+					destructor: function() {
+						var instance = this;
+
+						var toolbar = instance.get('toolbar');
+
+						toolbar.destroy();
+
+						instance.settingsForm.destroy();
```

Comment:
Hi, @brunobasto. The last @ctampoya's comment helped me to figure out the problem. The instance.settingsForm variable is only defined when the Sidebar is opened. In one of the Poshi test steps, it leaves the Form Builder page without open the Sidebar, so, this variable is never defined.

Here is my solution: https://github.com/fernandosouza/liferay-portal/blob/e80bc04d7f684e9f41b54ba1b40fc99cf357022f/modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/form_builder_field_settings_sidebar.js#L65

See the rebased commit:  https://github.com/brunobasto/liferay-portal/pull/720/commits/e80bc04d7f684e9f41b54ba1b40fc99cf357022f#L65